mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/builtin/impl/DefaultJavaNetURLBridge.java

```diff
@@ -30,9 +32,9 @@ protected String toString(URL value) {
 	@Override
 	protected URL fromString(String value) {
 		try {
-			return new URL( value );
+			return new URI( value ).toURL();
```

Comment:
I'm worried this could result in different behavior, since apparently `new URI` handles URL escapes while `new URL` does not.

On the other hand, it seems like the right thing to do, and I doubt many people index URL anyway.

Please mention this change in the migration guide?